content/docs/reference/usage/subcommands.md

```diff
@@ -38,6 +38,32 @@ Run the program, either directly on the host or in an emulated environment (depe
 ### flash
 Flash the program to a microcontroller.
 
+The `-monitor` flag can be given to the `flash` command to start the serial
+monitor (see below) immediately after flashing. However, some microcontrollers
+need a split second or two to configure its serial port after flashing, and
+using the `-monitor` flag can fail because the serial monitor starts too
+quickly. In that case, use the `tinygo monitor` command explicitly.
+
+### monitor
+Start the serial monitor program on the serial port that is connected to the
+microcontroller.
+
+If there are multiple microcontroller attached, use the `-port` flag to select
+the serial port. On Linux, the port will be something like `/dev/ttyUSB0` or
+`/dev/ttyACM1`. On MacOS, the port will look like `/dev/cu.usbserial-1420`.
```

Comment:
Looking at the source code, it appears you can actually use `tinygo monitor -target=xxx` to filter the ports. For many boards, TinyGo knows the USB vid/pid of the port and thus can pick the right port automatically.

Reply:
For USB-compatible microcontrollers, it is convenient to specify `-target=xxx` instead of `-port=xxx`.
However, when two identical microcontrollers are connected, for example, it is necessary to use `-port=xxx`.

USB-compatible microcontrollers at tinygo 0.27 : samd21, samd51, nrf52840, rp2040

Reply:
When there are multiple microcontrollers connected, the `tinygo monitor` prints an error message like this:
```
error: multiple serial ports available - use -port flag, available ports are /dev/ttyACM0, /dev/ttyACM1
```

I think it would be more clear to tell the user to use the `-port` flag as instructed by the error message. Please take a look at my wording of this in the updated commit.

Reply:
When running `tinygo flash`, `-target` is required, but `-port` is not.
I created `tinygo monitor` with the scenario that I would first try `tinygo monitor -target`, and if that could not connect, then I would use `-port`.

I don't know which is better, but it would be better to at least have it written that `-target=xxx` can be specified instead of `-port=yyy`.

Reply:
> When running `tinygo flash`, `-target` is required, but `-port` is not. I created `tinygo monitor` with the scenario that I would first try `tinygo monitor -target`, and if that could not connect, then I would use `-port`.
> 
> I don't know which is better, but it would be better to at least have it written that `-target=xxx` can be specified instead of `-port=yyy`.

I added explanations of the various `-port`, `-baudrate`, and `-target` flags on the `tinygo monitor` subcommand. They don't always work consistently. For example, on Arduino Zero clone (SAMD21, on `/dev/ttyACM0`), using just the `-target=arduino-zero` does not work if there is an Arduino Nano (`/dev/ttyUSB0`) attached.